### DISCUSSION TITLE
/framework/docs/providers/aws/guide/serverless.yml appears garbled

### DISCUSSION THREAD

**Discussion by mikeputnam** [serverless/serverless#11288] [General]
https://www.serverless.com/framework/docs/providers/aws/guide/serverless.yml

Seems like the documentation parser had a bad build?

**Comment by pgrzesik** [comment]
Hey @mikeputnam, thanks for reporting. We're aware of the problem and are working internally to resolve it. Here's an issue to track potential updates: https://github.com/serverless/serverless/discussions/11288

**Reply by mikeputnam** [reply]
Resolved. 👍
